niworkflows/interfaces/registration.py

```diff
@@ -372,17 +382,19 @@ class EstimateReferenceImage(SimpleInterface):
     output_spec = EstimateReferenceImageOutputSpec
 
     def _run_interface(self, runtime):
-        ref_name = self.inputs.in_file
+        # Select first EPI file
+        ref_name = self.inputs.in_file[0]
```

Comment:
This will select the first character of the input name if the `in_file` is only one path.
```suggestion
        ref_name = self.inputs.in_file
        if isinstance(self.inputs.in_file, list):
            ref_name = self.inputs.in_file[0]
```

Reply:
Oscar, I tested this one to be sure. You've got input and output behavior reversed.

Reply:
Gotcha. Then this inline `ensure_list` is unnecessary right? https://github.com/poldracklab/niworkflows/pull/408/files#diff-7a5a9f735d33c7c55dc21e359cdac718R137

Also, there's no `ensure_list` for the sbref input.

Reply:
I thought you needed to pass a list to a mapnode? And you don't need it for the sbref input because of the `InputMultiObject`.